go.mod

```diff
@@ -1,40 +1,38 @@
 module github.com/operator-framework/rukpak
 
-go 1.21
-
-toolchain go1.21.0
+go 1.22.0
 
 require (
 	github.com/containerd/containerd v1.7.17
 	github.com/go-git/go-billy/v5 v5.5.0
 	github.com/go-git/go-git/v5 v5.12.0
 	github.com/go-logr/logr v1.4.1
 	github.com/google/go-containerregistry v0.19.1
-	github.com/google/go-containerregistry/pkg/authn/k8schain v0.0.0-20240418155129-98dd3e91704f
-	github.com/google/go-containerregistry/pkg/authn/kubernetes v0.0.0-20240418155129-98dd3e91704f
+	github.com/google/go-containerregistry/pkg/authn/k8schain v0.0.0-20240505154900-ff385a972813
+	github.com/google/go-containerregistry/pkg/authn/kubernetes v0.0.0-20240505154900-ff385a972813
 	github.com/gorilla/handlers v1.5.2
 	github.com/nlepage/go-tarfs v1.2.1
 	github.com/onsi/ginkgo/v2 v2.17.3
 	github.com/onsi/gomega v1.33.1
-	github.com/operator-framework/api v0.23.0
-	github.com/operator-framework/helm-operator-plugins v0.2.1
-	github.com/operator-framework/operator-registry v1.40.0
+	github.com/operator-framework/api v0.24.0
+	github.com/operator-framework/helm-operator-plugins v0.2.2-0.20240520180534-f463c36fedf9
```

Comment:
Only reason I am pinning on a commit is because I wasn't sure if we wanted to cut a release of helm-operator-plugins since the Kubebuilder dependency was not updated to one that supports go 1.22 and k8s 1.30 for scaffolding. 

Happy to discuss this further (and cut a release if desired), but if we don't intend to cut a release for helm-operator-plugins in the very near future I'd prefer to get this in as is to unblock the CRD Upgrade Safety preflight check work

Reply:
Is `helm-operator-plugins` scaffolding still working right now with your changes that just merged? If so, I see no reason not to go ahead and tag it.

Reply:
I believe so. I can go ahead and tag it.

Reply:
As found in https://kubernetes.slack.com/archives/C0181L6JYQ2/p1716234751101299 the scaffolding is broken and should be fixed in helm-operator-plugins before being tagged. To unblock the in-flight CRD upgrade safety preflight check work we will capture fixing that in an issue to be addressed before tagging and will be keeping this pinned on a commit in the meantime